### PR TITLE
synDNA notebook update

### PR DESCRIPTION
**Pull request by RodolfoSalido**
added absolute quant functionality to the iSeqnorm notebook. The test data does not use the new functionality, but the samplesheet includes the new column.

### CODE REVIEW

File: metapool/tests/test_metapool.py
Comment (charles-cowart) [inline metapool/tests/test_metapool.py:1000]: It might be good to add a comment that this is specifically testing add_syndna without any parameters and we expect to see just an empty column.

File: metapool/tests/test_metapool.py
Comment (charles-cowart) [inline metapool/tests/test_metapool.py:1026]: Since syndna_pool_number and syndna_pool_number are only used once, it's usually preferred to not create new variables and just make the call:
test_df_ = add_syndna(test_df,syndna_pool_number='pool1', syndna_pool_concentration=2.35)

File: metapool/tests/test_metapool.py
Comment (charles-cowart) [inline metapool/tests/test_metapool.py:1059]: Similar to above, we can just use the values for these two variables directly in the parameters.

**Review by charles-cowart — CHANGES_REQUESTED** [review]
Looks good so far! I'll take a look at the notebooks next.

**Comment by charles-cowart** [comment]
@RodolfoSalido I think @mmbryant23 might want to review it at some point, but it looks good to me. The only thing I would ask to fix is the header numbering around the change, which is quite confusing. Here's the headers around the change, in order:

2.3. Step 3: calculate normalization volumes and add to plate map
## Step 3b (optional): reformat plate or add synDNA spike-in
2.5. Add synDNA spike-in
2.6. Step 4: make pick list
2.7. Step 5: write pick list to file

I think maybe changing Step 3b to be 2.4 might be an option? Or keeping it as-is and decrementing 2.5-2.7 by .1. Another thing I want to make sure of is that this notebook does several things, and Step is reset to 1 more than once. Perhaps Step 3b should be the start of a new Step 1?

**Comment by charles-cowart** [comment]
@RodolfoSalido One additional item: It looks like your tests are passing, but your code is not passing flake8 for style. The easiest way I know of to correct for this is to install flake8 into your environment with 'pip install flake8' and type 'flake8 metapool' on the command-line. This will bring up a list of line numbers and messages for everything flake8 doesn't agree with.

If it's a lot, you can type 'flake8 metapool | head' and this will limit it to the first ten; just make sure you run it again afterward to catch additional errors! :)

**Comment by RodolfoSalido** [comment]
I will work on improving the header numbering. I think we stopped paying close attention to it, but it would be beneficial to improve it to a state that makes sense. 

Will also work on flake8 styling fixes.

**Comment by charles-cowart** [comment]
> I will work on improving the header numbering. I think we stopped paying close attention to it, but it would be beneficial to improve it to a state that makes sense.
> 
> Will also work on flake8 styling fixes.

Thanks Rodolfo! :)

**Comment by RodolfoSalido** [comment]
While we wait for McKenzie to come back and review, I'll also consult with the wetlab to see if they have any feedback on the new modules.

**Review by charles-cowart — APPROVED** [review]
Approved! My only request is that the cells are cleared in the notebook before merging w/master. It might be better to do this after @mmbryant23 has reviewed it as well.

**Comment by charles-cowart** [comment]
@mmbryant23 @RodolfoSalido @antgonza Bumping to keep this PR in everyone's short-term memory.

**Review by mmbryant23 — APPROVED** [review]

**Pull request merged**
